fineract-provider/build.gradle

```diff
@@ -330,13 +330,32 @@ tasks.withType(JavaCompile) {
                 "UnusedNestedClass",
                 "UndefinedEquals",
                 "JdkObsolete",
-                "OperatorPrecedence"
+                "OperatorPrecedence",
+                "EmptyBlockTag",
+                "MissingSummary",
+                "MutablePublicArray",
+                "InvalidParam",
+                "ReturnFromVoid",
+                "AlmostJavadoc",
+                "PublicConstructorForAbstractClass",
+                "ProtectedMembersInFinalClass",
+                "StaticAssignmentInConstructor",
+                "UnescapedEntity",
+                "EmptyCatch",
```

Comment:
Seems this is not a valid check, maybe a typo or maybe you meant something else and wrote something else. I can merge this once this is resolved.